src/test/java/seedu/findvisor/logic/parser/ScheduleCommandParserTest.java

```diff
@@ -47,4 +49,18 @@ public void parse_invalidDatetimeFormat_throwsParseException() {
                 String.format(MESSAGE_INVALID_COMMAND_FORMAT, ScheduleCommand.MESSAGE_USAGE));
     }
 
+    @Test
+    public void parse_validRemark_returnsScheduleCommand() {
+        Meeting meeting = createValidMeetingNonEmptyRemark();
+        Index targetIndex = INDEX_FIRST_PERSON;
+        ScheduleCommand expectedScheduleCommand = new ScheduleCommand(targetIndex, meeting);
+        System.out.println(dateTimeToInputString(meeting.start) + " " + dateTimeToInputString(meeting.end));
```

Comment:
Is this line supposed to be included in the test?

Reply:
left traces of my print debugging😅